tests/effective_dps_test.cpp

```diff
@@ -527,6 +527,8 @@ static void check_knives( const std::function<Approx( const std::string & )> &ca
         CHECK( calc_expected_dps( "tanto" ) == 15.78 );
         CHECK( calc_expected_dps( "kris" ) == 14.08 );
         CHECK( calc_expected_dps( "knife_rambo" ) == 14.36 );
+        CHECK( calc_expected_dps( "knife_rm42" ) ==
+               40.51 ); //DPS is unusually high but this is a bigger, slower combat knife with 30% more cut damage.
```

Comment:
Please include the calculation and the "base" dps too. Also, 39/40 is still too high, it's over 25 before you apply the modifiers you mentioned. Please keep it at a cap of 20.

Reply:
I don't know what to say, its damage is literally a combat knife + CVD coating, *and* it is slower and heavier than a combat knife. I don't understand why its DPS is reporting as over 2x as high as a generic combat knife when it has total only 30% more stab damage and the same bash damage, and is slower.

Reply:
26 * 1.2 is 31, not 34, for a start. To-hit increases also have large effects on this metric.

Reply:
well, CVD is 1.3x damage, tempered steel is ~1.2x damage. I'm assuming that the combat knife is made of either tempered or high carbon steel so giving the RM42 knife identical damage to it, then x1.3 = 33.8 cut. I do think the DPS test probably overrates to hit a bit here, because there's no way +1 accuracy is accounting for over 50% more DPS compared to a CVD'd combat knife, or at least that shouldn't ever be the case in gameplay.

Reply:
> tempered steel is ~1.2x damage. I'm assuming that the combat knife is made of either tempered or high carbon steel so giving the RM42 knife identical damage to it, then x1.3 = 33.8 cut. 

This doesn't make sense to me.

Reply:
why doesn't it make sense to you?

Reply:
You're switching the multiplier.

Reply:
How so? For a given item, I'm assuming tempered steel is 20% stronger than "steel" (generic) as that's how Drew's weapons worked. CVD applies a 30% increase to cut damage. For the combat knife I'm assuming it's already made of good quality steel, so I'm skipping the 20% increase. Then, I'm adding the 30% more damage from the CVD to go from 26 to 34. As it is a bigger, heavier and longer blade, it should probably have more damage on top of that even.

Reply:
Oh, I see. Very sorry for the delayed reply.
The DIAMOND flags automatically applies the 30% increase, I thought you knew that and were just applying a 20% increase, as though it were made of better material than the other combat knife, but then it has a 30% increase.
By manually applying this 30% increase, you're double-dipping.

Reply:
Oh jeez, I'm so sorry about that. I could have sworn it added the 30% cut damage during the CVD process and the DIAMOND flag was just for the description. I will lower damage to 27, slightly above that of a combat knife, which should hopefully produce a more sane number. thanks for clearing that up!